feature_tests/src/result.rs

```diff
@@ -40,7 +40,7 @@ pub mod ffi {
             Err(())
         }
 
-        #[diplomat::attr(all(*, supports = fallible_constructors), named_constructor = "failing_struct")]
+        #[diplomat::attr(all(supports = named_constructors, supports = fallible_constructors), named_constructor = "failing_struct")]
```

Comment:
thought: we should support `auto` inside toplevel `all` even if we don't support it inside `any`